docs/build/ecosystem.md

```diff
@@ -38,12 +38,11 @@ The following document provides an ecosystem overview of how the various tools c
 
 |Company|Type of Quantum Computer|
 |--- |--- |
-|[Google QCS](https://quantumai.google/cirq/tutorials/google/start)|Superconducting qubits|
-|[Alpine Quantum Technologies](https://quantumai.google/cirq/tutorials/aqt/getting_started)|Trapped ions|
-|[IonQ](https://quantumai.google/cirq/tutorials/ionq/getting_started)|Trapped ions|
-|[Microsoft Azure Quantum](https://quantumai.google/cirq/tutorials/azure-quantum/getting_started_ionq)|Trapped ions (Honeywell and IonQ)|
-|[Pasqal](https://quantumai.google/cirq/tutorials/pasqal/getting_started)|Neutral atoms|
-|[Rigetti](https://quantumai.google/cirq/tutorials/rigetti/getting_started)|Superconducting qubits|
+|[Alpine Quantum Technologies](https://quantumai.google/cirq/hardware/aqt/getting_started)|Trapped ions|
```

Comment:
Did you delete the link to Google QCS here?  Was that intentional?

Reply:
Yes, since QCS is not available and the start page that is being linked to is un-indexed.